docs/rsdl/rapid-rsdl-capability-mapping.mdx

```diff
@@ -0,0 +1,221 @@
+| :------: | -------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+|   `eq`   | `SingleValue`                          | Compare a property and a constant value with the eq operator                                                                                                                              |
+|   `in`   | `MultiValue`                           | Compare a property with the in operator and a set of constant values                                                                                                                      |
+| `range`  | `SingleRange`                          | Compare a property with the open interval (i.e. `a < p and p < b`) where a and b are constants and p is the property. the `<` can be any of the comparison operators `<`, `<=`, `>`, `>=` |
+| `ranges` | `MultiRange`                           | One of more `range` expressions combined by or                                                                                                                                            |
+| `prefix` | a specialization of `SearchExpression` | Compare a property with a constant using the startsWith function                                                                                                                          |
+|  `text`  | `SearchExpression`                     | Compare a property with a constant using any of the string comparison functions startsWith, contains, endsWith                                                                            |
+|   -/-    | `MultiRangeOrSearchExpression`         |                                                                                                                                                                                           |
+|  `any`   | no restrictions                        | Any combination of the above using `and`, `or`, and `not` and any of the built-in functions                                                                                               |
+
+## Expansion
+
+```rsdl
+path /orders {
+    GET {
+        expand {
+            items {
+                expand {
+                    sku
+                }
+            }
+        }
+    }
+}
+```
+
+```xml
+<Annotations Target="orders">
+  <Annotation Term="Org.OData.Capabilities.V1.ExpandRestrictions">
+    <Record Type="Org.OData.Capabilities.V1.ExpandRestrictionsType">
+      <PropertyValue Property="Expandable" Bool="true" />
+    </Record>
+  </Annotation>
+</Annotations>
+<Annotations Target="orders/items">
+  <Annotation Term="Org.OData.Capabilities.V1.ExpandRestrictions">
+    <Record Type="Org.OData.Capabilities.V1.ExpandRestrictionsType">
+      <PropertyValue Property="Expandable" Bool="true" />
+    </Record>
+  </Annotation>
+</Annotations>
+
+```
```

Comment:
Again we'll have to express the expand _restrictions_, and the new target syntax is not (yet) recognized by any tools.

What the OpenAPI generator already recognizes is this syntax:
https://github.com/oasis-tcs/odata-openapi/blob/15220b9a26a2e6c40ad4dd57e1712f275d23b08d/test/csdl2openapi.test.js#L1983-L1990